ts/Dashboard/EditMode/Sidebar.ts

```diff
@@ -939,6 +969,23 @@ class Sidebar {
                     (updatedSettings as any)[fieldId] = JSON.parse(
                         (formFields[i] as HTMLTextAreaElement).value
                     );
+
+                    if (
+                        fieldId === 'chartOptions' &&
+                        (updatedSettings as any).chartOptions &&
```

Comment:
Looks like this evaluates to false currently. Might be due to chartOptions being before the chart type in the order?